### ISSUE TITLE
Calling GINSIM from R

### ISSUE THREAD

**Issue by caramirezal** [caramirezal/SQUAD#2]
The following tutorial explains how to call java scripts from R.

[https://darrenjw.wordpress.com/2011/01/01/calling-java-code-from-r/](url)

Additionally, to run the the last script calling ginsim on the terminal system is necessary to compile and execute with the following commands.

 // compilling
> javac -cp "ginsim.jar" testGINSIM.java 

// running
> java -cp ".:ginsim.jar" testGINSIM

Where ginsim.jar is the jar file downloaded from the ginsim project web page.

**Comment by caramirezal** [comment]
In order to use the rJava R library java most be set properly with the following commands:

> /usr/lib/jvm/java-8-oracle/jre/lib/amd64
> /usr/lib/jvm/java-8-oracle/jre/lib/amd64/server

See the next reference

https://stackoverflow.com/questions/12872699/error-unable-to-load-installed-packages-just-now/25932828#25932828?newreg=355546cf7dd64fa2a26de0ac0ad3fdeb

And the installing the library in the terminal using the next line

> apt-get install r-cran-rjava

**Comment by caramirezal** [comment]
I decided to use the system command to call the GINSIM java methods. I used this form instead of rJava() mainly because it is necessary to adjust the pathway to the "GINSIM environment", that is to use the command  "javac -cp" when compiling and executing the method. Even if it is possible, I did not figure out how to do it using rJava(). The way I solved it is shown in the GetAttractors.java and getAttractorsAsynchronous.R files.
